deltachat-rpc-client/tests/test_securejoin.py

```diff
@@ -224,6 +224,7 @@ def check_account(ac, contact, inviter_side, please_wait_info_msg=False):
     # Start second Bob device, if it wasn't started already.
     bob2.start_io()
     bob2.wait_for_securejoin_joiner_success()
+    bob2.wait_for_event(EventType.CHATLIST_CHANGED)
```

Comment:
It's weird that this is only needed for the `bob2` device, the reason is that events arrive in another order on the second device. But i'm not sure it's worth complicating the test and trying to wait for this event for the first device somehow. Maybe it's better then to fix Core and emit CHATLIST_CHANGED one more time after SecureJoin finishes